### PR TITLE
What happens after remediation

### LINKED ISSUE #76: Incorporate "fix" vs "mitigate" into SSVC decision process
The "fix" actions, once completed, should close the vulnerability management case and not invite recursion. This makes applying a patch qualitatively different from completing a remediation. It's unclear the best way to explain or notate this.

 - [ ] Decide on most accessible method of incorporating "fix" vs "mitigate" into the SSVC decision process.

(Related to issue #46)

### PR DESCRIPTION
**Pull request by j---**
fixes #76 

I broke up the text Chuck produced because I thought it improved flow.  

It seems like some white space got cleaned up and caused a bunch of spurious diffs. The only actual changes are in `### Deploying Patches `

### CODE REVIEW

File: doc/md_src_files/040_stakeholders-scope.md [hunk 1]
```diff
@@ -116,16 +116,21 @@ Table 2:<a name="table-supplier-outcomes"></a> Proposed Meaning for Supplier Pri
...
 
-<!--**Talk about applying other mitigations here** TODO
--->
+As opposed to mitigation, applying a remediation finishes an SSVC analysis of a deployed system.
+While specific vulnerabilities in specific systems can be remediated, the vulnerability cannot be 'disposed of' or eliminated from future consideration within an IT environment.
+Since software and systems are dynamic, a single vulnerability can be re-introduced after initial remediation through updates, software rollbacks, or other systemic actions that change the operating conditions within an environment.
+It is therefore important to continually monitor remediated systems for re-introduced vulnerabilities even after initial remediation.
```
Comment (ahouseholder) [inline doc/md_src_files/040_stakeholders-scope.md:133]: Should this chunk be talking about both remediated and newly deployed systems? I think you've covered the case where

1. deployed system is remediated
2. vul is reintroduced to deployed system

but not 
1. vul is remediated in deployed systems
2. new system deployed with vul

File: doc/md_src_files/040_stakeholders-scope.md [hunk 2]
```diff
@@ -116,16 +116,21 @@ Table 2:<a name="table-supplier-outcomes"></a> Proposed Meaning for Supplier Pri
 
 ### Deploying Patches
 
-Whether or not to deploy available remediation is a binary decision. However, additional defensive mitigations may be necessary sooner than patches are available and may be advisable after patches are applied. We recognize that vulnerability management actions are different when a patch is available and when it is not available.
+Whether or not to deploy available remediation is a binary decision because remediation, when successful, eliminates a vulnerability in a specific system.
```
Comment (ahouseholder) [inline doc/md_src_files/040_stakeholders-scope.md:119]: deploy _an_ available remediation

File: doc/md_src_files/040_stakeholders-scope.md [hunk 3]
```diff
@@ -116,16 +116,21 @@ Table 2:<a name="table-supplier-outcomes"></a> Proposed Meaning for Supplier Pri
...
+However, additional defensive mitigations may be necessary sooner than patches are available and may be advisable after patches are applied.
+Mitigation, from a tactical perspective, is an attempt to block or minimize risk to a vulnerable system without actually eliminating the vulnerability.
+We recognize that vulnerability management actions are different when a patch is available and when it is not available.
```
Comment (ahouseholder) [inline doc/md_src_files/040_stakeholders-scope.md:122]: Should this replace patch etc with remediation for consistency?

**Review by ahouseholder — CHANGES_REQUESTED** [review]
I had a couple suggestions

File: doc/md_src_files/040_stakeholders-scope.md (hunk 3 above)
Comment (j---) [reply doc/md_src_files/040_stakeholders-scope.md]: https://github.com/CERTCC/SSVC/pull/128/commits/4b2d110e101a9c6e0138c1fc1066e8a2316626c7

File: doc/md_src_files/040_stakeholders-scope.md (hunk 2 above)
Comment (j---) [reply doc/md_src_files/040_stakeholders-scope.md]: https://github.com/CERTCC/SSVC/pull/128/commits/4b2d110e101a9c6e0138c1fc1066e8a2316626c7

File: doc/md_src_files/040_stakeholders-scope.md (hunk 1 above)
Comment (j---) [reply doc/md_src_files/040_stakeholders-scope.md]: https://github.com/CERTCC/SSVC/pull/128/commits/4b2d110e101a9c6e0138c1fc1066e8a2316626c7

**Review by ahouseholder — APPROVED** [review]
This looks good to me as of 4b2d110e101a9c6e0138c1fc1066e8a2316626c7

File: doc/md_src_files/040_stakeholders-scope.md (hunk 1 above)
Comment (laurie-tyz) [reply doc/md_src_files/040_stakeholders-scope.md]: Consider deleting this part of the sentence:

'disposed of' or

Comment (laurie-tyz) [reply doc/md_src_files/040_stakeholders-scope.md]: regarding the "new system deployed with vul"  

Deploying a new system with the same vul should be detected by other mitigations.  (should) continuous scanning; auto patch deployment; configuration management -- one of these should identify the new system with the same (now old) vulnerability that was remediated.   Adding a new system puts us at the beginning of the decision tree once the vul is identified. Adding a new system with the same vul shouldn't happen, but it does.

**Review by laurie-tyz — APPROVED** [review]
looks good.

**Comment by cgyarbrough** [comment]
The changes look good to me as well.

**Pull request merged**